docs/ai_actions/ai_actions.md

```diff
@@ -13,6 +13,5 @@ You can also extend it to perform other tasks or support additional AI services.
 
 [[= cards([
 "ai_actions/ai_actions_guide",
-"ai_actions/install_ai_actions",
-"ai_actions/extend_ai_actions"
+"ai_actions/install_ai_actions"
 ], columns=4) =]]
```

Comment:
Does it need an update?
```suggestion
], columns=2) =]]
```

Reply:
The value does not change anything, and as more tiles are added, it will help format the page properly. Our default setting is 4.